manifests/charts/istio-control/istio-discovery/templates/poddisruptionbudget.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.global.defaultPodDisruptionBudget.enabled }}
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
I think this has already been updated for a while. Make sure to rebase the branch.